kube/utils.go

```diff
@@ -116,9 +116,12 @@ func newTypeMeta(apiVersion, kind string, modifiers ...helm.NodeModifier) *helm.
 	return mapping
 }
 
-func newObjectMeta(name string) *helm.Mapping {
+func newObjectMeta(name string, annotations *map[string]string) *helm.Mapping {
```

Comment:
Annotations don't make sense for most types of objects; I think it would be better to leave it out of `newObjectMeta` and just have it in `newKubeConfig`. Or maybe even leave it out of that, and do it in the callers.

Reply:
Ok, added it just for `jobs` for now, which is the only place we need annotations AFAIK.

Rebased on master and pushed again.